country_pilots/japan/summary.html

```diff
@@ -15,7 +15,7 @@
 </p>
 
 <p>
-  As part of the biomass harmonization collaboration, JAXA provides the global ALOS (for the time period 2007-2010) and ALOS-2 (2015-present) radar imagery that constitute the key input data for the CCI Biomass, JPL, and NCEO Africa biomass products.
+  As part of the biomass harmonization collaboration, JAXA provides the global ALOS (for the time period 2007-2010) and ALOS-2 (2015-present) radar imagery that constitute the key input data for the CCI Biomass and NCEO Africa biomass products.
```

Comment:
not sure it's necessary to remove hear, but easy enough to add back later (if someone remembers)